src/App/styles.less

```diff
@@ -40,6 +40,8 @@
     --modal-background-color: rgba(15, 13, 32, 1);
     --outer-glow: 0px 0px 30px rgba(123, 91, 245, 0.37);
     --border-radius: 0.75rem;
+
+    background-color: #161523;
```

Comment:
I copied this value from the `WebpackPwaManifest` config in `webpack.config.js`, so since it has to be hard-coded there, I thought it'd make sense to do the same here.
I've added a comment about that to make it more clear.

This ensures the title bar color is consistence across devices and browsers, instead of fallbacking to each browser's behavior which are inconsistent.